graphene_pydantic/converters.py

```diff
@@ -266,6 +266,10 @@ def convert_generic_python_type(
         return convert_union_type(
             type_, field, registry, parent_type=parent_type, model=model
         )
+    elif origin == T.Literal:
```

Comment:
Might be a good idea to add a check that the Literal type exists for the python version being used